example/styleguide.json

```diff
@@ -1,5 +1,9 @@
 {
   "title": "React Style Guide",
+  "typekit": {
+    "id": "rtq7sdx",
```

Comment:
At this moment the only documentation available for the `styleguide.json`is this file. I will improve it later, but for now I'm just building on the existing system.